<launch>
  <arg name="launch_prefix" default="" /> <!-- set to value="gdbserver localhost:10000" for remote debugging -->
  <arg name="node_namespace" default="tagslam_ros_node" />

  <!-- load other parameters from yaml-->
  <rosparam command="load" file="$(find tagslam_ros)/config/test.yaml" ns="$(arg node_namespace)" />

  <node pkg="tagslam_ros" type="tagslam_ros_test_node" name="$(arg node_namespace)" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)" />
</launch>
